accounts-journey/src/main/kotlin/com/backbase/accounts_journey/common/Mapper.kt

```diff
@@ -0,0 +1,15 @@
+package com.backbase.accounts_journey.common
+
+/**
+ *  Try catch a mapping.
```

Comment:
Why is this needed? Why silence all exceptions?

Reply:
This is similar as `runCatching`, but then not returning Kotlin.Result. This is a safe way of mapping. I can add a log in case the mapping fails.